Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2400,6 +2400,7 @@ Color RenderStyle::colorResolvingCurrentColor(CSSPropertyID colorProperty, bool
 {
     auto result = unresolvedColorForProperty(colorProperty, visitedLink);
 
+    // FIXME: not sure we really need this block?
```

Comment:
I think this block is incorrect, there is actually a WPT testing we never apply the stroke color on text-decoration. So I think we should remove it in a separate patch (but there might be other code to remove as well, which is why I don't want this to be done here).

Reply:
Yes I will try to remove it in another patch.

Reply:
Can we drop this comment or change it?

Reply:
It's a good reminder we should do a follow-up patch

Reply:
I think we want to change this comment though.

> FIXME: not sure we really need this block?

It's clear what the answer is to this comment, so can we reword it?

> // FIXME: This block looks incorrect (webkit.org/b/...).